themes/default/content/blog/pulumi-release-notes-80/index.md

```diff
@@ -0,0 +1,281 @@
+Destroying (demo/dev)
+```
+
+👉  Learn more in the [Add --remove as an option for `pulumi destroy` GitHub issue](https://github.com/pulumi/pulumi/issues/10484).
+
+### Add Output.format to the Python SDK
+
+We added Output.format to the Python SDK to match helper methods in our other SDKs. Most languages have a standard way to format a string (`String.Format` in C#, `fmt.Sprintf` in Go, etc) but you can't use those functions with our `Output` values. We have added a format method that's similar to each language's normal format method, but with support for outputs. We have added `Output.format` in Python which matches the [standard Python `str.format` method](https://docs.python.org/3/library/stdtypes.html#str.format). This is similar to how we have `interpolate` in TypeScript, and `Output.Format` in C#.
+
+👉  Learn more in the [Add Output.format to python SDK pull request](https://github.com/pulumi/pulumi/pull/10919).
+
+### Cloning Pulumi templates from Azure DevOps
+
+We now support cloning Pulumi templates (`pulumi new`) from Azure DevOps repositories. When given a URL with a hostname of `dev.azure.com`, it will use system-installed git and will clone the master/main branch.
+
+### Skip Checkpoints Experimental Flag
+
+We have added a new experimental flag to improve performance for certain use cases. This flag makes Pulumi skip saving [state checkpoints](/docs/intro/concepts/state/#checkpoints) as it modifies resources and instead only save the final state of a deployment. This is an experimental feature that also requires an PULUMI_EXPERIMENTAL=true environment variable to be set. Using the feature introduces risk that in the case of network disconnect or crash state edits will be lost and may require manual recovery. When this risk is acceptable, using the feature can speed up Pulumi deployment times. As it is in experimental we may make changes to how it has been implemented down the track.
+
+👉 Learn more by reviewing the [Skip checkpoints pull request](https://github.com/pulumi/pulumi/pull/10750).
+
+### Token Authentication in Go Providers
+
+Thank you to community member [@avlllo](https://github.com/avlllo) for adding support for token authentication in the Go providers that use git. You can now authenticated requests by tokens such as `https://auth:token@gitlab.example.com/group/proj.git/tree/v0.0.0/path`.
+
+👉 Learn more by reviewing the [Add token to git Go module pull request](https://github.com/pulumi/pulumi/pull/10628).
+
+### Bump TypeScript version from v3.7.3 to v3.8.3
+
+We updated the vendored version of TypeScript in the NodeJS SDK and runtime from v.3.7.3 to v3.8.3. Updating TypeScript versions enables you to use newer TypeScript features, such as the `import type` and` export type` syntax, which is required to use OpenTelemetry.
+
+👉 Learn more by reviewing the [Bump TSC version to 3.8.3 pull request](https://github.com/pulumi/pulumi/pull/10618).
+
+### Display outputs last in diff view
```

Comment:
This one is honestly just a bug fix - it really just highlights that we used to do something that makes no sense :-).  I'd be fine leaving it out (especially since it is visually prominent due to the size of the code blocks).  But don't feel strongly - it is indeed a nice fix.